db/migrate/20200513170154_add_user_id_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserIdToPosts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :posts, :user_id, :integer
```

Comment:
It is better to add this column foreign key. This ensures there is DB Level constraint to this column.
You can read about `references` in rails migration.

Reply:
Makes sense, a foreign key would give us better ACID properties. Still learning how exactly models/migrations work. Thank you for the review.

Reply:
Will push this change in the next commit.